mobile-widgets/qmlmanager.cpp

```diff
@@ -315,7 +329,7 @@ void QMLManager::finishSetup()
 		int error = parse_file(existing_filename);
 		if (error) {
 			// we got an error loading the local file
-			appendTextToLog(QString("got error %2 when parsing file %1").arg(existing_filename, get_error_string()));
+			appendTextToLog(QString("got error %2 when parsing file %1").arg(existing_filename, m_lastError));
```

Comment:
`appendTextToLog` is probably called twice here given `parse_file()` already does a `report_error()`?
do we actually need the extra error string here?

also the `parse_file()` errors strings are translated.

Reply:
Yes, re-logging the error message is redundant. This could be replaced by
```
appendTextToLog(QString("Error parsing file %1").arg(existing_filename))
```
I don't think translating is worth it, since this will only end up in the debug log, which a normal user won't see.

Reply:
i think `parse_file()` will report the error that it cannot parse `existing_filename`, then `appendTextToLog()` will report the the error again. i don't think we need `appendTextToLog` here.